sdk/python/feast/expediagroup/vectordb/milvus_online_store.py

```diff
@@ -91,9 +92,15 @@ def online_write_batch(
         ],
         progress: Optional[Callable[[int], Any]],
     ) -> None:
-        raise NotImplementedError(
-            "to be implemented in https://jira.expedia.biz/browse/EAPC-7971"
-        )
+        with MilvusConnectionManager(config.online_store):
+            try:
+                entities = self._format_data_for_milvus(data)
+                collection_to_load_data = Collection(table.name)
+                collection_to_load_data.insert(entities)
+                #  The flush call will seal any remaining segments and send them for indexing
+                collection_to_load_data.flush()
+            except Exception as e:
```

Comment:
I feel that we shouldn't put a try / except around the entire code and rather catch errors more targeted where they actually occur. We can address it later, but if you see ways to do it now please do so.

Reply:
Sure